### DISCUSSION TITLE
Add a list of strings using the repeater interface

### DISCUSSION THREAD

**Discussion by jacobsandersen** [directus/directus#7198] [Feature Requests]
Forgive me if this already exists but I have been poking around and cannot find a way to do this easily.

I am looking to use the repeater interface to add a list of strings, so that the resulting API is simply a JSON array of strings as opposed to a JSON array of objects with inner fields that are strings.

To see what I'm talking about, try this:

- make a new field on a model of type JSON
- go to interface and select repeater
- create new, type string, but do not enter a field name
- press create, then go to display and set type to raw
- continue and finish making field
- go to item detail
- press create new on the field and it'll add a field with no data in it at all and no way to edit the string

The only way to do what I want successfully is to simply set the interface as code and write JSON. That's fine, but not as user friendly as I personally would want.

It would be great to have this! Thanks for all your hard work

**Comment by rijkvanzanten** [comment]
Next to "Tags", there's currently no "repeater style" input for adding a list of strings

**Reply by benhaynes** [reply]
I think you could do this as CSV, right? Not what OP is looking for, but changing the datatype is another option.

**Reply by rijkvanzanten** [reply]
Tags works with both the CSV or JSON type 👍🏻

**Reply by jacobsandersen** [reply]
Hi guys, thanks for the reply. Sorry, I may be misunderstanding but were you suggesting me to do something specific next to "Tags"? Or, were you agreeing that this doesn't yet exist? Or (number 3 :P), are you saying to try using the Tags interface rather than code? This could work, but might look a bit weird for longer strings.

Specific use-case here is for job highlights which are typically a one or more sentences long.

Example of "looking weird":
![img](https://i.imgur.com/7Z7RC6i.png)

But yes, that does create a JSON list! For now though I think I'll keep writing the JSON myself. Thanks for the reply again!

**Reply by rijkvanzanten** [reply]
I mostly mentioned tags as a reference for ourselves like "that does create a JSON list!" 😄 Not ideal for a list of longer sentences for sure, but it's an interface that uses the same IO 🙂

**Reply by jacobsandersen** [reply]
Gotcha! Haha, thanks :D

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
